test/fixtures/missing-css/x/component/component.js

```diff
@@ -0,0 +1,6 @@
+import { LightningElement } from 'lwc'
+import stylesheet from './stylesheet.css'
+
+export default class extends LightningElement {
+    static stylesheets = [stylesheet]
```

Comment:
This actually doesn't do anything unless we update the LWC deps, but for the purpose of this test I don't think it matters.